python/adelphi/README.md

```diff
@@ -0,0 +1,107 @@
+
+    adelphi --keyspaces foo,bar export-gemini
+
+And to store these schemas in a directory "baz":
+
+    adelphi --keyspaces foo,bar --output-dir=baz export-gemini
+
+### contribute
+This command automates the workflow of contributing one or more schemas to the Adelphi project.  The [Adelphi schema repository](https://github.com/datastax/adelphi-schemas) is implemented as a Github repository and contributions to this repository take the form of pull requests.  The workflow implemented by this command includes the following steps:
+
+* Fork the Adelphi schema repository into the Github workspace for the specified user
+** If the user has already forked the schema repository that fork will be re-used
+* Create a branch in the forked repository
+* Extract schemas from the specified Cassandra instance
+* Add files representing the contents of these schemas to the branch in the forked repsitory
+* Create a pull request on the Adelphi schema repository for the newly-created branch and files
+
+The syntax for using this command looks very similar to the export commands above.  The following will create a pull request to contribute schemas for the keyspaces "foo" and "bar" to Adelphi:
+
+    adelphi --keyspaces foo,bar contribute
+
+Authentication to Github is performed by way of a [personal access token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).  You must create a token for your Github user before you can contribute your schema(s) to Adelphi.  The token can be provided to the command at execution time using a command-line argument but this is discouraged for security reasons.  Instead we recommend using an environment variable, in this case the **ADELPHI_CONTRIBUTE_TOKEN** environment variable.  We discuss using environment variables to pass command-line arguments in the "Options" section below.
+
+## Options
+The "adelphi" application supports several command-line arguments.  The full list of arguments can be accessed via the following:
+
+    adelphi --help
+
+The output of this command provides a brief summary of each argument:
+
+    $ adelphi --help
+    Usage: adelphi [OPTIONS] COMMAND [ARGS]...
+    
+    Options:
+      --hosts TEXT                  Comma-separated list of contact points
+                                    [default: 127.0.0.1]
+    
+      --port INTEGER                Database RPC port  [default: 9042]
+      --username TEXT               Database username
+      --password TEXT               Database password
+      --keyspaces TEXT              Comma-separated list of keyspaces to include.
+                                    If not specified all non-system keypaces will
+                                    be included
+    
+      --rf INTEGER                  Replication factor to override original
+                                    setting. Optional.
+    
+      --anonymize / --no-anonymize  Enable/disable schema anonymization  [default:
```

Comment:
Here I don't think that noting the default as True makes it clear which is the default since it's sort of an either/or situation, not a single flag with a boolean variable.

Reply:
Fair point.  This is a weirdness of how the Python CLI arg lib we're using handles things.  When you declare a boolean flag it adds two CLI args, one for the name and one as "no-" name.  The flag is then turned on or off based on which one you add, so there is something of a decoupling there.

I think I'll change the default to something like "anonymization is enabled" to make that clearer.

Reply:
Okay, this now looks like...

```
...
  --anonymize / --no-anonymize  Enable/disable schema anonymization  [default:
                                anonymization enabled]
...
```

To me that's clearer all around.  Whaddya think @jdonenine ?